srcs/ConfigFile/ConfigFile.hpp

```diff
@@ -32,6 +32,7 @@ public :
 		Path				mErrorPage;		// def = error.html
 		std::vector<std::string> mMethod;		// def = GET HEAD PUT POST DELETE OPTIONS
 		std::vector<std::string> mCgiProgramPath; // def = .bla
```

Comment:
```suggestion
		std::vector<std::string> mCgiProgramPath; // def = ft_tester/bla_tester, /usr/bin/php
```

"/usr/bin/php");	this-&gt;mCgiProgramPath.push_back("ft_tester/bla_tester");